config/vite.json

```diff
@@ -0,0 +1,17 @@
+{
+  "all": {
+    "sourceCodeDir": "app/frontend",
+    "watchAdditionalPaths": []
+  },
+  "development": {
+    "host": "www.school.localhost",
```

Comment:
I've removed this setting and updated `content_security_policy.rb` to allow the default `localhost:3036` as well. Removing this is okay since `vite-ruby` settings can also be controlled via environment variables.